molo/profiles/templates/profiles/editprofile.html

```diff
@@ -6,15 +6,41 @@
 <div class="profiles">
 	<form method="post" action="{% url "molo.profiles:edit_my_profile" %}" class="profiles__form">
 	{% csrf_token %}
```

Comment:
@Ntu2koTrevor please add the check `{% if settings.profiles.UserProfilesSettings.activate_display_name %}` here since we made the default True